.github/scripts/end2end/deploy-metadata.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -exu
+
+. "$(dirname $0)/common.sh"
+
+# create a separate namespace for metadata
+kubectl create namespace metadata
+
+# clone the metadata repository
+git init metadata
+cd metadata
+git fetch --depth 1 --no-tags https://${GIT_ACCESS_TOKEN}@github.com/scality/metadata.git
+git checkout FETCH_HEAD
```

Comment:
Do we want to support a specific metadata version? What if the latest commit of the default branch introduced a breaking change or a bug/limitation to be fixed in another PR before a release is made?

Reply:
probably not worth it:
* this chart is mostly not maintained
* it references "unknown" version of metadata
* ...consequently this is not a long term solution, just a stop-gap measure until we switch to using metadata-operator to deploy a "recent" metadata : but at least it allows us to test OOB already...

Reply:
No problem, I'm fine if that's not that maintained